### DISCUSSION TITLE
Enable session affinity with Aspire

### DISCUSSION THREAD

**Discussion by mip1983** [microsoft/aspire#6796] [Q&A] [answered]
Is there a way to enable session affinity (for signalR) on the AppHost project? Is it something you can do via 'PublishAsAzureContainerApp' or configuring the endpoints?

**Comment by davidfowl** [comment, marked as answer]
You would do it via PublishAsAzureContainerApp:

```C#
PublishAsAzureContainerApp((module, app) =>
{
    app.Configuration.Ingress.StickySessionsAffinity = StickySessionAffinity.Sticky;
});
```
